README.md

```diff
@@ -4,6 +4,35 @@ This repository contains a CMake template for Archicad Add-On Development.
 
 [![Build](https://github.com/GRAPHISOFT/archicad-addon-cmake/actions/workflows/build.yml/badge.svg)](https://github.com/GRAPHISOFT/archicad-addon-cmake/actions/workflows/build.yml)
 
+## Cloning the repository
```

Comment:
I would put this section under the Prerequisites section.

Reply:
Resolved